staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -120,7 +120,8 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringSliceVar(&s.WatchCacheSizes, "watch-cache-sizes", s.WatchCacheSizes, ""+
 		"List of watch cache sizes for every resource (pods, nodes, etc.), comma separated. "+
 		"The individual override format: resource#size, where size is a number. It takes effect "+
```

Comment:
The admin needs to know how a "resource" is identified (since this is done in different ways in different parts of k8s).

Reply:
added more description to make the format clearer. PTAL